ci/qs-test/pipeline.vars.yml

```diff
@@ -19,6 +19,12 @@ publish-bucket:
   prefix: public/main/
 
 ci-bucket:
+  #! Because we use regional buckets, `taskcat upload` will ensure to create
```

Comment:
I am not sure, why we need to put such constraints.
We are not experts in taskcat usage; however, we should not add more assumptions, which will be difficult to remember.

Reply:
I am not following. This is for us to work with the taskcat config we have in the root of our repo, and making our CI work properly with that. So that this taskcat config can be used by both us and AWS.